03-setup-ingress-controller/ambassador.md

```diff
@@ -528,10 +528,43 @@ A `L4` load balancer replaces the original `client IP` with its `own IP` address
 
 To enable proxy protocol for `AES`, you need to:
 
-1. Enable `proxy` protocol on `DigitalOcean` LB through service `annotation` on `Ambassador` LB service.
-2. Enable `proxy` protocol configuration on `Ambassador Edge Stack` end.
+- Enable `proxy` protocol on `DigitalOcean` LB through service `annotation` on `Ambassador` LB service.
 
-For different `DigitalOcean` load balancer configurations, please refer to the examples from the official [DigitalOcean Cloud Controller Manager](https://github.com/digitalocean/digitalocean-cloud-controller-manager/tree/master/docs/controllers/services/examples) documentation. Proxy protocol on the `LoadBalancer` is enabled with the following `annotation` on `Ambassador` LB service: `service.beta.kubernetes.io/do-loadbalancer-enable-proxy-protocol: "true"`. You must **NOT** create a load balancer with `Proxy` support by using the `DigitalOcean` web console, as any setting done outside `DOKS` is automatically `overridden` by DOKS `reconciliation`.
+Edit the `Helm` values file provided in the `Starter Kit` repository using an editor of your choice (preferably with `YAML` lint support). For example, you can use [VS Code](https://visualstudio.microsoft.com):
+
+```shell
+code 03-setup-ingress-controller/assets/manifests/ambassador-values-v6.7.13.yaml
+```
+
+Then, uncomment the `service` section as seen below:
+
+```yaml
+service:
+  type: LoadBalancer
+  annotations:
+    # Enable proxy protocol
+    service.beta.kubernetes.io/do-loadbalancer-enable-proxy-protocol: "true"
+    # Specify whether the DigitalOcean Load Balancer should pass encrypted data to backend droplets
+    service.beta.kubernetes.io/do-loadbalancer-tls-passthrough: "true"
+```
+
+Finally, after saving the values file, you can apply changes using `Helm`:
+
+```shell
+HELM_CHART_VERSION="6.7.13"
+
+helm upgrade ambassador datawire/ambassador --version "$HELM_CHART_VERSION" \
+ --namespace ambassador  \
+ -f "03-setup-ingress-controller/assets/manifests/ambassador-values-v${HELM_CHART_VERSION}.yaml"
+```
+
+For different `DigitalOcean` load balancer configurations, please refer to the examples from the official [DigitalOcean Cloud Controller Manager](https://github.com/digitalocean/digitalocean-cloud-controller-manager/tree/master/docs/controllers/services/examples) documentation. Proxy protocol on the `LoadBalancer` is enabled with the following `annotation` on `Ambassador` LB service: `service.beta.kubernetes.io/do-loadbalancer-enable-proxy-protocol: "true"`.
+
+  **Note:**
+
+You must **NOT** create a load balancer with `Proxy` support by using the `DigitalOcean` web console, as any setting done outside `DOKS` is automatically `overridden` by DOKS `reconciliation`.
+
+- Enable `proxy` protocol configuration on `Ambassador Edge Stack` end.
 
```

Comment:
After applying the changes via Helm, I would add a small `curl` example showing the user that proxy protocol is enabled and it works. It means, the client should see own IP address in the response.